### PR TITLE
Added live example for 'Adding Regions' doc

### PR DESCRIPTION
**Pull request by CharlyJazz**
### Proposed changes
 - The live example shows a practical scenario where regions are added to the view depending on the model

Link to the issue: https://github.com/marionettejs/backbone.marionette/issues/3209

### CODE REVIEW

File: docs/marionette.region.md [hunk 1]
```diff
@@ -233,6 +235,8 @@ myView.addRegions({
 });
 ```
 
+[Live example](http://jsfiddle.net/charlyjazz/gze5ykov/49/)
```
Comment (scott-w) [inline docs/marionette.region.md]: Does this need to be switched out for a `marionettejs` account URL?

**Comment by paulfalgout** [comment]
@scott-w we'll have to swap out the link at merge.  

As far as the router is concerned since `AppRouter` is being made an external component this is currently the easiest way to keep this from needing to be updated with v4.  We certainly could change it to use the other lib at that time, but it also doesn't seem necessary.

**Comment by cusial** [comment]
Unsubscribe me to this thread please…


Best Regards,

NW Computing Office Manager


NW Computing Office Manager
        [cid:logo_horizontal_inverted_60cb0183-c447-43a8-a150-00e8baf7caa5.png]


Level 12, 459 Little Collins St, Melbourne VIC 3000
Level 15, 1 Castlereagh St, Sydney NSW 2000
Level 22, 127 Creek Street, Brisbane QLD 4000
Level 13, 50 Cavill Ave, Surfers Paradise QLD 4217

P 1300 484 529
E Office.Manager@nwcomputing.com.au<mailto:Office.Manager@nwcomputing.com.au> | W nwcomputing.com.au<http://nwcomputing.com.au>

[cid:fb-icon_6f360223-2e9d-47de-95f8-16d578342f26.gif]<http://facebook.com/nwcomputing> [cid:tw-icon_944dcbc2-146a-40b6-ae6c-988cc5bf9473.gif] <http://twitter.com/nwcomputing>  [cid:in-icon_17ac0afa-8c93-45da-a314-230ae0c839e7.gif] <https://www.linkedin.com/company/nw-computing>

[cid:MSPMentor_20b72b30-fb78-4ed4-9a32-b6f514990acf.png]

**Comment by paulfalgout** [comment]
@cusial to my knowledge this is something you need to do yourself.  There should be an unsubscribe button on this PR and you can unwatch the library at the top of github.  There's also an unsubscribe link _in the email_.  But we don't have the ability to do this for you.

**Comment by CharlyJazz** [comment]
Friends missing something to do here?

File: docs/marionette.region.md (hunk 1 above)
Comment (blikblum) [reply docs/marionette.region.md]: I've forked at http://jsfiddle.net/marionettejs/kjvzdyd6/ to use marionettejs account

**Comment by blikblum** [comment]
@CharlyJazz after updating the last jsfiddle link i think is good to go

**Comment by paulfalgout** [comment]
I'll get this in after https://github.com/marionettejs/backbone.marionette/pull/3561 lands

**Comment by CharlyJazz** [comment]
Hahaha what?

**Comment by CharlyJazz** [comment]
It's been a long time since I have a beard :tada:

**Comment by paulfalgout** [comment]
Added to `next`

**Pull request closed without merging**